Week2/homework.js

```diff
@@ -0,0 +1,109 @@
+const prompt = require('prompt');
+const util = require('util');
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
```

Comment:
You should use hyfuser and hyfpassword as credentials, otherwise, the program doesn't have enough privileges to log on to the MYSQL server